features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/slowframes/SlowFramesListener.kt

```diff
@@ -105,7 +104,7 @@ internal class DefaultSlowFramesListener(
     @WorkerThread
     override fun onAddLongTask(durationNs: Long) {
         val view = currentViewId
-        if (durationNs >= anrDuration && view != null) {
+        if (durationNs >= configuration.anrDuration && view != null) {
```

Comment:
I think there is a mix of terminology here, because long task is not exactly the same as ANR. Should we align wording?

Reply:
The Idea is to start consider long tass as an ANR after some threshold. changed naming a bit

Reply:
Maybe we need to put it away for now? We have already 2 ways to report ANRs:

* reading `ApplicationExitInfo` to report fatal ANRs - this is enabled by default
* using Watchdog approach to report non-fatal ANRs - this is disabled by default, because it is noisy

Here it is also similar to the latter approach, so it may bring a lot of noise and also adds one more reporting channel. And it may bring inconsistency if customer compares number of ANRs reported as errors (the formers ones, fatal ones) with number of views having this attribute.

And as I said, using a single threshold for ANR doesn't really work, because from the system point of view, it is different for different cases.